editoast/openapi.yaml

```diff
@@ -8372,11 +8378,8 @@ components:
         locked:
           type: boolean
           description: Whether the schedule item is locked (only for display purposes)
-        on_stop_signal:
-          type: boolean
-          description: |-
-            Whether the next signal is expected to be blocking while stopping
-            Can be true only if `stop_for` is `Some`
+        reception_signal:
+          $ref: '#/components/schemas/ReceptionSignal'
```

Comment:
The description is not forwarded, I tried to dig different solutions, any idea is welcome.